services/app-api/utils/dynamo/dynamodb-lib.ts

```diff
@@ -9,6 +9,8 @@ import {
   QueryCommandInput,
   PutCommand,
   PutCommandInput,
+  ScanCommandInput,
+  ScanCommand,
```

Comment:
Of course, if you choose to do that, you'll need to import it.

```suggestion
  paginateScan,
```